### PR TITLE
Add IT technician

### PR DESCRIPTION
**Pull request by gabrielamatias**
@petehamilton , @jesssummerfield : please review.

### CODE REVIEW

**Comment by petehamilton** [comment]
Code looks 👍 but branch needs rebasing on master. Good to amend the commit and explain what's going on in the commit as well.

File: app/pages/about/jobs/positions/IT-technician.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          our desire to build high-quality systems and processes regardless of which department they work in.
+        </p>
+        <p className='para'>
+          We pride ourselves on providing our customers with the best product and support possible. We’re looking for
+          someone to help build the IT platforms and processes necessary to enable everyone in the company to do just that.
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          You’ll be responsible for keeping GoCardless’ employees up and running on a day-to-day basis.
+          ou’ll work with all departments within the company, and will have the chance to own and collaborate on
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/IT-technician.js]: @gabrielamatias 'You'll' not 'ou'll'

File: app/pages/about/jobs/positions/IT-technician.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            Procuring, managing, provisioning, and decommissioning employee Macs.
+          </li>
+          <li>
+            Backing up employee laptops.
+          </li>
+          <li>
+          Enforcing IT security policies, such as laptop disk encryption and password policies.
+          </li>
+        </ul>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>what we're looking for</h4>
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/IT-technician.js]: @gabrielamatias should be What not what (I think it's a small w you've used but might just be my eyes...)

**Comment by jesssummerfield** [comment]
@gabrielamatias two tiny changes but all good apart from that!

**Pull request merged**